OpenRA.Mods.Common/Warheads/CreateEffectWarhead.cs

```diff
@@ -44,15 +44,9 @@ public class CreateEffectWarhead : Warhead
 		[Desc("Chance of impact sound to play.")]
 		public readonly int ImpactSoundChance = 100;
 
-		[Desc("Consider explosion above this altitude an air explosion.",
-			"If that's the case, this warhead will consider the explosion position to have the 'Air' TargetType (in addition to any nearby actor's TargetTypes).")]
-		public readonly WDist AirThreshold = new WDist(128);
-
 		[Desc("Whether to consider actors in determining whether the explosion should happen. If false, only terrain will be considered.")]
 		public readonly bool ImpactActors = true;
 
-		static readonly BitSet<TargetableType> TargetTypeAir = new BitSet<TargetableType>("Air");
```

Comment:
This is still only used from this one warhead. How many other places are expected to use this in the future?

Reply:
Not sure (possibly no upstream ones), but I wasn't sure about not moving this along with `AirThreshold`.

But I guess we can leave it here until we come up with more uses for it.